bin/radical-pilot-create-static-ve

```diff
@@ -187,18 +191,18 @@ then
 fi
 
 echo -n "create  virtualenv "
-stdbuf -oL $VIRTENV_CMD -p "python$VERSION" "$PREFIX" | progress
+stdbuf -oL $VIRTENV_CMD --no-site-packages -p "python$VERSION" "$PREFIX" | progress
```

Comment:
with this one I have an error: `virtualenv: error: unrecognized arguments: --no-site-packages`, it seems like this option was dropped, skip it?